src/renderer/components/reader/ReaderApp.tsx

```diff
@@ -92,24 +82,62 @@ const queryParams = getURLQueryParams();
 const computeReadiumCssJsonMessage = (): IEventPayload_R2_EVENT_READIUMCSS => {
     const store = (container.get("store") as Store<any>);
     const settings = store.getState().reader.config;
-    const cssJson = {
-        align: settings.align,
-        colCount: settings.colCount,
-        dark: settings.dark,
+
+    // TODO: see the readiumCSSDefaults values below, replace with readium-desktop's own
```

Comment:
Much more info in the code comments, and of course in the original ReadiumCSS doc (which is linked too): https://github.com/readium/r2-navigator-js/blob/develop/src/electron/common/readium-css-settings.ts